src/platform/utilities/environment/va-subdomain.js

```diff
@@ -5,6 +5,6 @@
  */
 export default function isVATeamSiteSubdomain() {
   return !!window.location.hostname.match(
-    /(cem|choose|benefits|telehealth|volunteer|ptsd|nutrition).va/gi,
+    /(benefits|cem|choose|ethics|healthquality|hepatitis|hiv|mentalhealth|move|nutrition|oedca|oefoif|oprm|osp|patientcare|patientsafety|polytrauma|ptsd|publichealth|ruralhealth|sci|telehealth|vetcenter|volunteer).va/gi,
```

Comment:
we should try to generate this from the config instead of hard coding it here.

Reply:
I totally agree